### DISCUSSION TITLE
Tuya Dimmer missing Clusters for Binding

### DISCUSSION THREAD

**Discussion by roteRakete66** [Koenkk/zigbee2mqtt#23847] [Devices]
I have this device [Bseed ZigBee Dimmer](https://www.bseed.com/products/bseed-eu-russia-new-zigbee-touch-wifi-light-dimmer-smart-switch?variant=42116733534363), which became device support from this request [Tuya Dimmer](https://github.com/Koenkk/zigbee2mqtt/issues/21980) and it doesn't work entirely as expected.
1. This issue is not very important to me, but I cannot control the backlight.
2. I have several of these devices and would like to bind them, but it seems the corresponding clusters for turning on/off and adjusting brightness are missing.

Here is the database entry for one of them:
```
{"id":7,"type":"Router","ieeeAddr":"0xa4c138e4c3cb7ec4","nwkAddr":64523,"manufId":4417,"manufName":"_TZE204_vevc4c6g",
"powerSource":"Mains (single phase)","modelId":"TS0601","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,
"devId":81,"inClusterList":[4,5,61184,0],"outClusterList":[25,10],"clusters":{"genBasic":{"attributes":{"65506":56,
"65508":0,"65534":0,"modelId":"TS0601","manufacturerName":"_TZE204_vevc4c6g","powerSource":1,"zclVersion":3,
"appVersion":74,"stackVersion":0,"hwVersion":1,"dateCode":""}}},"binds":[],"configuredReportings":[],"meta":{}},
"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],
"configuredReportings":[],"meta":{}}},"appVersion":74,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,
"interviewCompleted":true,"meta":{"configured":-903893911},"lastSeen":1725452438719}
```

If someone could help me, that would be great.
If you need more information, feel free to ask.

**Comment by Koenkk** [comment]
Tuya does not follow the Zigbee spec, it doesnt implement the onOff cluster so you cannot bind it.

**Reply by roteRakete66** [reply]
Sad to hear that. Then I guess I’ll have to handle it in Home Assistant through an automation. Thanks anyway.

**Comment by G87007** [comment]
![Screenshot 2025-03-11 195002](https://github.com/user-attachments/assets/ecbbc328-812c-48e7-8361-dd52a34b717c)
![Screenshot 2025-03-11 194919](https://github.com/user-attachments/assets/e41c1afa-965a-4708-9122-56ea259b8338)
I also have similar issues, all my clusters is missing and I m not able to bind any devices, the picture is for one device but all of them show me same options

**Reply by MSL-DA** [reply]
Did you read
> Tuya does not follow the Zigbee spec, it doesnt implement the onOff cluster so you cannot bind it.

**Comment by G87007** [comment]
what other dimmer switch can you recommend for bind ?
